pkg/chartutil/validate_name.go

```diff
@@ -39,11 +39,11 @@ var (
 	errMissingName = errors.New("no name provided")
 
 	// errInvalidName indicates that an invalid release name was provided
-	errInvalidName = errors.New("invalid release name, must match regex ^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])+$ and the length must not longer than 53")
+	errInvalidName = errors.New("invalid release name, must match regex ^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])+$ and the length must not be longer than 53")
```

Comment:
Don't the uppercase letters need to be removed from the regex here as well?

Reply:
Shoot, meant to add that also when updating it. Good catch, thanks

Reply:
Updated